docs/dev-instance.md

```diff
@@ -16,3 +16,19 @@ docker exec kimai2 rm /opt/kimai/var/data/kimai.sqlite
 docker exec kimai2 /opt/kimai/bin/console kimai:reset-dev
 docker exec kimai2 /opt/kimai/bin/console kimai:create-user admin admin@example.com ROLE_SUPER_ADMIN changemeplease
 ```
+
+## The bleading edge
```

Comment:
`bleading edge` or `bleeding edge` - no seriously: no joke, I have no clue. Just wondering because the translators accept both but seem to prefer "bleeding"

Reply:
Doesn't really matter, but bleading is a joke on leading + bleeding = bleading

;)

Reply:
One of these situations where I don't feel the language enough, but it sounds logical.
Quest solved for today: learn something new ;-)

Reply:
Believe me Kevin I wish my [German|French|Spanish|Romanian|Mandarin|etc] was as good anyone I meet who has English as their second language. :)